### PR TITLE
Explicitly filter out Sanic warnings after Sanic is initialized

### PR DESCRIPTION
**Pull request by aleksandarmijat**
**Proposed changes**:
- Explicitly filter out Sanic warnings after Sanic is initialized

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation in the [rasaHQ/rasa](https://github.com/rasaHQ/rasa)
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

**Review by ancalita — APPROVED** [review]

**Comment by Tawakalt** [comment]
Do we need a changelog?

**Comment by radovanZRasa** [comment]
We should at list have a misc changelog.

File: changelog/1120.misc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Added the functionality that resets the Sanic warnings filter, which was allowing the triggering of Sanic warnings.
```
Comment (Tawakalt) [inline changelog/1120.misc.md:1]: Reset Sanic warnings filter to prevent the triggering of Sanic warnings.

Comment (Tawakalt) [inline changelog/1120.misc.md:1]:
```suggestion
Reset Sanic warnings filter to prevent the triggering of Sanic warnings.
```

**Review by Tawakalt — APPROVED** [review]

**Pull request merged**